codalab/apps/web/tasks.py

```diff
@@ -834,6 +834,7 @@ def do_phase_migrations():
     logger.info("Checking {} competitions for phase migrations.".format(len(competitions)))
     for c in competitions:
         c.check_future_phase_sumbmissions()
+    logger.info("@@@ Done checking phase migrations!")
```

Comment:
Do we need the `@@@` in this log statement?

Reply:
We don't need it, for sure, but definitely helps out when skimming the log